### PR TITLE
pipeline: outputs: kinesis: update doc for time_key_format.

### PR DESCRIPTION
**Pull request by Claych**
Signed-off-by: Clay Cheng <claychen@amazon.com>

### CODE REVIEW

**Comment by PettitWesley** [comment]
@Claych please add spaces in commit and title: pipeline:outputs:firehose:update

File: pipeline/outputs/kinesis.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
 | region | The AWS region. |
 | stream | The name of the Kinesis Streams Delivery stream that you want log records sent to. |
 | time\_key | Add the timestamp to the record under this key. By default the timestamp from Fluent Bit will not be added to records sent to Kinesis. |
-| time\_key\_format | strftime compliant format string for the timestamp; for example, the default is '%Y-%m-%dT%H:%M:%S'. This option is used with time\_key. |
+| time\_key\_format | strftime compliant format string for the timestamp; for example, the default is '%Y-%m-%dT%H:%M:%S'. Now can support millisecond with '%3N' and support nanosecond with '%9N' and '%L'; for example, adding '%3N' to support millisecond '%Y-%m-%dT%H:%M:%S:%3N'. This option is used with time\_key. |
```
Comment (PettitWesley) [inline pipeline/outputs/kinesis.md:22]: Docs are timeless, so do not use words like "now". Just "Supports millisecond with..."

Comment (PettitWesley) [reply pipeline/outputs/kinesis.md:22]: > '%Y-%m-%dT%H:%M:%S:%3N'

Should it be `.%3N` since that is what most users would configure?

Comment (Claych) [reply pipeline/outputs/kinesis.md:22]: sure, will delete "now" and change ":" to "."

File: pipeline/outputs/kinesis.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
 | region | The AWS region. |
 | stream | The name of the Kinesis Streams Delivery stream that you want log records sent to. |
 | time\_key | Add the timestamp to the record under this key. By default the timestamp from Fluent Bit will not be added to records sent to Kinesis. |
-| time\_key\_format | strftime compliant format string for the timestamp; for example, the default is '%Y-%m-%dT%H:%M:%S'. This option is used with time\_key. |
+| time\_key\_format | strftime compliant format string for the timestamp; for example, the default is '%Y-%m-%dT%H:%M:%S'.Support millisecond with '%3N' and support nanosecond with '%9N' and '%L'; for example, adding '%3N' to support millisecond '%Y-%m-%dT%H:%M:%S.%3N'. This option is used with time\_key. |
```
Comment (PettitWesley) [inline pipeline/outputs/kinesis.md:22]: grammar/language nit: I believe it reads better as "Supports millisecond precision '%3N' and supports nanosecond precision with..."

**Review by PettitWesley — APPROVED** [review]

**Pull request merged**